### DISCUSSION TITLE
How to change stackamount Label font size and wierd behavior with hovering if u hold item

### DISCUSSION THREAD

**Discussion by Kristo-Art** [peter-kish/gloot#195] [Q&A] [answered]
Hello,

First I want to thank you for this brilliant add-on, big appreciation for your hard work!

I want to ask for away to change text Label font size displaying stack amount.
I tried to change it in some places like where label is created but it doesn't affect display.

Also i found wierd behavior in CtrlInventoryGrid if u change field_dimensions or item_spacing with hovering when holding item in right and top sides of panel and if u change theme this goes wierder, i didnt dig yet but maybe someone found a solution for that?

Thanks in advance!

**Comment by peter-kish** [comment, marked as answer]
Unfortunately, the only way to do this in the current version is on a theme level. You would have to assign a new theme to the inventory control and override the default label font.

Defining themes for custom UI controls still has [no official workflow in Godot 4.2](https://github.com/godotengine/godot-cpp/issues/1332#issuecomment-1852792776), but [this comment](https://github.com/godotengine/godot-cpp/issues/1332#issuecomment-2041060614) from only a week ago suggests a workaround. I will give it a try, hopefully it doesn't introduce any side effects.

**Comment by Kristo-Art** [comment]
Thanks for your quick answer!

My workarround was to change default theme scale and custom font in Editor settings/GUI.

Do you have any advice how to fix this hovering with item wierd behavior?

**Reply by peter-kish** [reply]
Oh sorry, missed that part...
But I'm not sure if I understand what exactly the issue is. Can you give a bit more information? Or even better, if you have an example scene I can take a look at, that would help a lot.

**Reply by Kristo-Art** [reply]
This issue is in example also.

There is issue with hovering when u drag item to the edge of top side or left side it doesnt always hover empty slot where u want to drop it  

This works perfectly without item and with item at bottom or right.

Example: Just open Inventory_Grid_stacked_ex take an item and move arround at top or left sides 
![test](https://github.com/peter-kish/gloot/assets/78514969/2e667252-5b57-4e34-b827-91504040e619)

**Reply by peter-kish** [reply]
This is how it behaves for me:
![Peek 2024-04-21 12-59](https://github.com/peter-kish/gloot/assets/11254996/3d96d556-5215-4708-b94b-1bd7a6b48398)
Note that in some cases the drop fails because the mouse was released outside the inventory control.
